packages/x-components/src/x-modules/facets/components/facets/facets.vue

```diff
@@ -18,9 +18,9 @@
             @binding {Facet} facet - Facet to render
       -->
       <slot
-        v-if="$scopedSlots[facetId]"
+        v-if="$scopedSlots[slotName(facetId)]"
         v-bind="{ facet, selectedFilters: selectedFiltersByFacet[facetId] || [] }"
-        :name="facetId"
+        :name="slotName(facetId)"
```

Comment:
Why not just map the items like you did previously, and avoid calling twice the  wrongly named `slotName` method from inside the render function?